modules/gapi/doc/00-root.markdown

```diff
@@ -17,7 +17,7 @@ is volatile at the moment and there may be minor but
 compatibility-breaking changes in the future.
 
 # Contents
-
+gapi
 G-API documentation is organized into the following chapters:
```

Comment:
something wrong here.

Reply:
Same issue on 4.x, I'll add a fix for it to the #25162